ccd-definition/AuthorisationCaseField/systemUpdateGAspec.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "CaseFieldID": "generalApplications",
+    "UserRole": "caseworker-civil-systemupdate",
+    "CRUD": "CRUD"
+  },
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "CaseFieldID": "generalAppType",
+    "UserRole": "caseworker-civil-systemupdate",
+    "CRUD": "CRUD"
+  },
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "CaseFieldID": "generalAppPBADetails",
+    "UserRole": "caseworker-civil-systemupdate",
+    "CRUD": "CRUD"
+  },
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
```

Comment:
There are some missing fields. eg. generalAppEvidenceDocument

Reply:
Please check in this PR - https://github.com/hmcts/civil-ccd-definition/pull/369/files

Reply:
These will need to be added later by @smitatalawadekar-hmcts .